holoviews/element/raster.py

```diff
@@ -396,6 +397,8 @@ def __init__(self, data, extents=None, **params):
 
 
     def _compute_raster(self):
+        if issubclass(self.interface, GridInterface):
+            return self, np.flipud(self.dimension_values(2, flat=False))
```

Comment:
Perhaps all interfaces could have a property to check to see if this return can be done directly instead of using ``isinstance``?